### DISCUSSION TITLE
Is this limited to country to create the National ID?

### DISCUSSION THREAD

**Discussion by ghost** [mosip/mosip#3] [Q&A] [answered]
Is this project limited to country to create the National ID such that all different IDs such as PAN, Voter ID, Employee ID, PF ID, Student ID etc eventually become this one ID? 

Or

Can this be used to create ID of any kind  (ex: Global employee ID, Global Student ID, Global visitors ID, Global Customer ID, Global Caller ID etc.while also generate virtual ID per country, ad-hoc, company if need be) by private companies or organisations which could either be derived from National ID card or does not need derive now but later be able to link to National ID Card when government implements them in their country?

**Comment by gsasikumar** [comment, marked as answer]
Yes, you can this is not limited to the country id.
